packages/teleterm/src/main.ts

```diff
@@ -55,6 +60,16 @@ app.whenReady().then(() => {
     });
   }
 
+  const isWin = process.platform === 'win32';
```

Comment:
You should be able to read platform from `mainProcess.settings`. Long term I think we should stick to a single way of obtaining platform details.